loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -84,4 +94,25 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
         return parameters;
     }
 
+    private static void assertLessThanOrEqualToReferenceVersion(String tag, String version, String referenceVersion) {
+        if (version == null) {
+            throw new PowsyblException("LoadflowParameters. Unexpected null pointer for version");
+        } else if (version.compareTo(referenceVersion) > 0) {
```

Comment:
Rather than doing that, you could simply call `Objects.requireNonNull(version)`.

Reply:
Done